addons/mrp/views/mrp_production_views.xml

```diff
@@ -239,12 +239,14 @@
                                         </tree>
                                     </field>
 
+                                    <field name="propagate_cancel" invisible="1"/>
```

Comment:
Relevant in this PR ?

Reply:
Yep, those 3 fields are in the `_prepare_merge_moves_distinct_fields`, we prepared the values for them but since they are not in the view, their values are not stored. But when adding new components through the tablet view, the moves are created directly without the form. So the new moves are considered to be different from the original moves.

Reply:
ok thanks :+1: